### ISSUE TITLE
Missing navigation panel in site editor and empty states

### ISSUE THREAD

**Issue by SaxonF** [WordPress/gutenberg#48906]
## What problem does this address?
On a fresh install when no menu's exist the navigation panel is missing from the site editor sidebar. This creates an inconsistent/unpredictable sidebar experience and doesn't allow for guidance if a user _wants_ to add a menu to their site but doesn't know how. (Navigation panel is natural place for this).

## What is your proposed solution?
We should always show the navigation link/panel. As to what we show in it, a few options depending on time constraints:

In the immediate short term, we could just extend description to include  "Get started by adding a navigation block to your site.".
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/1072756/223628429-5da23c96-10a4-452c-83a7-d220b37cbfbe.png">

In the mid term, we could show a page list block which should also be default state of the navigation block if no menu is there so users can still browse their site.

In the long term, if we do decide to move in the direction of including an "unlinked pages" section in the navigation panel, any pages would be listed there and we'd include a shortcut to adding a navigation block to your site's header.

**Comment by jameskoster** [comment]
I don't know how helpful the empty Navigation panel is, if it contains no actions 🤔 

For a site with no menus saved, it would probably be good to simply list all the pages and their structure, similar to how the Page List block appears here. This would match the behavior of the Navigation block itself. If we could allow re-ordering by updating the page `order` as well that may even be adequate for the majority of sites.

**Comment by SaxonF** [comment]
@jameskoster that was the mid term option except it would create menu and use page list block behind the scenes. It sounds like [this](https://github.com/WordPress/gutenberg/pull/48625) is doing something similar. Question is whether we have time for 6.2?

**Comment by jameskoster** [comment]
Oh yes, I was thinking beyond 6.2.

For 6.2 I don't know how useful the empty panel would be. 

> Get started by adding a navigation block to your site

This action alone will not be adequate I think? My understanding was that the Navigation block will default to displaying Page List by default, and not save a menu unless you change something, or add a block like Site Logo. If that's the case then the Navigation panel would continue to be hidden.

**Comment by talldan** [comment]
The way the navigation item loads in asynchronously is a bit of an issue too. Several times I've gone to click on 'Templates' after loading browse mode, and 'Navigation' appeared under my mouse cursor and I incorrectly clicked on that.

https://user-images.githubusercontent.com/677833/223897664-7bc88903-8295-4cf4-8e45-13921f2d74f2.mp4

**Comment by Mamaduka** [comment]
I agree; "popcorning" the "Navigation" panel isn't great UX. Always displaying the panel with the "getting started" message would be more helpful.

**Comment by draganescu** [comment]
@SaxonF is this and #50483 the same thing?

**Comment by annezazu** [comment]
Closing this out as https://github.com/WordPress/gutenberg/pull/50321 was merged previously and this is no longer relevant.
